src/config/consts.js

```diff
@@ -2,6 +2,42 @@ const ONCE_A_DAY = 'ONCE_A_DAY';
 const ONCE_PER_HOUR = 'ONCE_PER_HOUR';
 
 export const consts = {
+  a11yLabel: {
+    address: 'Adresse',
+    backIcon: 'Zurück Taste',
+    backIconHint: 'Navigieren zurück zur vorherigen Seite',
+    bookmarkList: 'Lesezeichenliste (Taste)',
+    bookmarkListHint: 'Zu der Lesezeichenliste hinzufügen',
+    button: 'Taste',
+    category: 'Kategorie',
+    closeMenuIcon: 'Schließen Taste',
+    closeMenuHint: 'Menü schließen',
+    heading: 'Überschrift',
+    imageCarousel: 'Bild des Bildkarussells',
+    imageCarouselLinkHint: 'Navigiert zum Gegenstand',
+    infoProvider: 'Anbieterinformationen (Taste)',
+    lunch: 'Mittagstisch(Gerichtname):',
+    mail: 'E-mail adresse',
+    mailHint: 'Wechselt zur E-Mail-App',
+    mapHint: 'Wechselt zur Karten-App',
+    openMenuHint: 'Navigiert zum Menü',
+    openMenuIcon: 'Menü Taste',
```

Comment:
(Taste) needs brackets

Reply:
solved in 62c6f5c318a48af248f0b6bc0380987d343bbce5

Reply:
I don't see this change anywhere.

Reply:
corrected in ef58ac7be4e00799d2a955770af47c10df993eed